app/controllers/application_controller.rb

```diff
@@ -1,4 +1,4 @@
 # frozen_string_literal: true
 
-class ApplicationController < ActionController::API
+class ApplicationController < ActionController::Base
```

Comment:
this is okay for now. Later we'll create a new ApiController for all your api controllers.